api/v2/views/access_token.py

```diff
@@ -0,0 +1,45 @@
+from rest_framework import status
+from rest_framework.response import Response
+
+from core.models import AccessToken, AtmosphereUser
+from core.models.access_token import create_access_token
+from core.query import only_current_access_tokens
+
+from api.exceptions import invalid_auth
+from api.v2.serializers.details import AccessTokenSerializer
+from api.v2.views.base import AuthModelViewSet
+
+class AccessTokenViewSet(AuthModelViewSet):
+
+    """
+    API endpoint that allows AccessTokens to be viewed or edited.
+    """
+    serializer_class = AccessTokenSerializer
+
+    def get_queryset(self):
+        """
+        Filter projects by current user.
+        """
+        user = self.request.user
+        qs = AccessToken.objects.filter(token__user=user)
+        if 'archived' in self.request.query_params:
+            return qs
+        # Return current results
+        return qs.filter(only_current_access_tokens())
+
+    def create(self, request):
+        issuer_backend = request.session.get('_auth_user_backend', '').split('.')[-1]
```

Comment:
Curious, why are you looking this up in the session?

Reply:
Issuer may just as well be "Personal-Access-Token"

Reply:
It is the only way I knew how to get the issuer by copying what is done in [`auth.py`](https://github.com/cyverse/atmosphere/blob/468a1cc3254efb0a793a4254112d6e3acf20c518/api/auth.py#L50-L53)

Reply:
In this case you're the issuer. Do some searching for where we pass values for issuer in d-c-a, which should show its purpose.

Reply:
The session is a key value store which is used in web-apps to track a user across multiple stateless requests. It's a bag of properties. If a user submits requests and passes their session as a cookie, we'll know which session they're using. `_auth_user_backend` is probably some relic of long ago

Reply:
Ok, I will look into that